### PR TITLE
LPS-178078 Extract

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js [hunk 1]
```diff
@@ -209,7 +211,74 @@ export function MenuItem({item, onMenuItemRemoved}) {
...
 								<ClayLayout.ContentCol gutters>
-									<ClayIcon symbol="drag" />
+									<ClayButtonWithIcon
+										displayType="unstyled"
+										monospaced={false}
+										onBlur={() =>
+											setIsMovementEnabled(false)
+										}
+										onKeyDown={(event) => {
+											if (event.key === 'Enter') {
```
Comment (sandrodw3) [inline modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js:229]: Did you think about moving the onKeyDown logic to a function? 😄 At least for me, the code looks a bit more readable when we don't have long callbacks inline in the listeners

**Review by sandrodw3 — COMMENTED** [review]
This is awesome 🤍 the code looks super clean and beautiful as usual 😄 I'm leaving some functional suggestions:
- Could we use for the cards that are being moved, the same styles that we have when dragging with mouse? I know the figma does not specify it but I think it would make the interaction more clear when starting the movement and also when stopping it
- When the sidebar is closed, it's being opened when we press Enter on a drag handler to start the movement, is this intended?

I'm adding the "Changes Required" label but feel free to forward if we don't want to implement these changes 😄

**Comment by ealonso** [comment]
Hey @victorg1991 I think the test results are related with your changes

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**